src/Storage/Session.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace PhpAb\Storage;
+
+use InvalidArgumentException;
+
+/**
+ * Stores the participation state of the user in a session.
+ */
+class Session implements StorageInterface
+{
+    /**
+     * @var string The namespace of the session.
+     */
+    private $namespace;
+
+    /**
+     * Initializes a new instance of this class.
+     *
+     * @param string $namespace The namespace of the session.
+     */
+    public function __construct($namespace)
+    {
+        if (!$namespace) {
+            throw new InvalidArgumentException('The namespace is invalid.');
+        }
+
+        $this->namespace = $namespace;
+    }
+
+    /**
+     * Gets the namespace that is used for this stroage.
+     */
+    public function getNamespace()
+    {
+        return $this->namespace;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function has($identifier)
+    {
+        if (empty($_SESSION[$this->getNamespace()])) {
+            return false;
+        }
+
+        return !empty($_SESSION[$this->getNamespace()][$identifier]);
```

Comment:
use isset in this method. Edge cases could be (string) "0". But I'm not sure if it is allowed in the session